javaslang/src/test/java/javaslang/collection/LinkedHashMultimapTest.java

```diff
@@ -94,12 +96,59 @@ protected String className() {
     }
 
     @Override
-    protected <K extends Comparable<K>, V> Multimap<K, V> mapOf(K key, V value) {
-        return LinkedHashMultimap.withSeq().of(key, value);
+    protected <K extends Comparable<K>, V> LinkedHashMultimap<K, V> mapOf(K key, V value) {
+        switch (containerType) {
+            case SEQ:
+                return LinkedHashMultimap.withSeq().of(key, value);
+            case SET:
+                return LinkedHashMultimap.withSet().of(key, value);
+            case SORTED_SET:
+                return LinkedHashMultimap.withSortedSet(naturalComparator()).of(key, value);
+        }
+        throw new RuntimeException();
```

Comment:
Does not matter much but I would have created a `default:` case for the throwing case... maybe a matter of taste